src/openrct2/peep/Staff.cpp

```diff
@@ -489,6 +492,8 @@ static uint8_t staff_handyman_direction_to_nearest_litter(Peep* peep)
     nextTile.y = (peep->y & 0xFFE0) + CoordsDirectionDelta[nextDirection].y;
 
     tileElement = map_get_first_element_at(nextTile.x / 32, nextTile.y / 32);
+    if (tileElement == nullptr)
+        return nextDirection;
```

Comment:
`return 0xFF;` The  path to the litter is on an invalid tile in this situation so we should not try to path towards it. This might cause some really odd things to happen in almost every case though so I guess it doesn't ultimately matter what is done.